frontend/src/components/events/index.css

```diff
@@ -14,3 +15,43 @@ tr {
   margin: 10px;
   padding: 90px;
 }
+
+.eventImage {
+  max-width: 240px;
+  max-height: 210px;
+}
+
+.eventbright-results {
+  margin: 25px;
+  align-content: center;
+}
+
+.event_container {
+  margin: 20px;
+}
+.event_container img {
+  margin-bottom: 30px;
+}
+
+.btn {
+  font-size: 15px;
+  font-weight: bold;
+  color: white;
+  background-color: green;
+  border-radius: 2px;
+  padding-right: 25px;
+  padding-left: 25px;
```

Comment:
just a heads up. You could potentially do this with some shorthand `padding: 10px 25px` -> this would set the top/bottom as 10px and the right/left as 25px as well